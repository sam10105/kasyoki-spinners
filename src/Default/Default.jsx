import React from 'react';
import './style.css';

const Default = () => (
  <div className="default">
    <div />
    <div />
    <div />
    <div />
  </div>
);

export default Default;
