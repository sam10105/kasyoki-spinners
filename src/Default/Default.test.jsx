import React from 'react';
import { render } from '@testing-library/react';
import Default from './Default';

it('should take a snapshot', () => {
  const { asFragment } = render(<Default />);

  expect(asFragment(<Default />)).toMatchSnapshot();
});
