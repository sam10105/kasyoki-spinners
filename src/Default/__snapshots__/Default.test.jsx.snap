// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="default"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</DocumentFragment>
`;
